src/autopas/containers/verletClusterLists/ClusterTower.h

```diff
@@ -139,7 +135,10 @@ class ClusterTower : public ParticleCell<Particle> {
    * @return
    */
   std::vector<Particle> &&collectAllActualParticles() {
-    _particles._particles.resize(getNumActualParticles());
+    if (not _particles._particles.empty()) {
+      // workaround to remove requirement of default constructible particles.
```

Comment:
mention that this always shrinks and therefore `value` is never actually inserted